e2e/test/scenarios/dashboard-filters/dashboard-filters-sql-number.cy.spec.js

```diff
@@ -58,15 +58,13 @@ describe("scenarios > dashboard > filters > SQL > text/category", () => {
     );
   });
 
-  it(`should work when set as the default filter`, () => {
+  it("should work when set as the default filter", () => {
```

Comment:
Do you it'd be a good idea to prohibit using template tags unnecessarily? I believe it's this [`quote`](https://eslint.org/docs/latest/rules/quotes) rule. That doesn't have to be fixe in this PR though.

Reply:
I have a PR for it, but no time to discuss/deliver it https://github.com/metabase/metabase/pull/32471